tests/plugins/colordialog/manual/showcurrentcolor.md

```diff
@@ -0,0 +1,29 @@
+@bender-tags: feature, 4.12.0, 2639
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog
+
+**Scenario 1:**
+
+1. Select text in editor 1.
+
+2. Click `Text Colour` button and `More Colours` button.
+    #### Expected
+    Color of text is by default focused, highlighted, selected and its RGB code is displayed.
+
+3. Select different color and click `OK` button.
+    #### Expected
+    Text color has been changed. Text is selected.
+
+4. Click `Text Colour` button and `More Colours` button again.
+    #### Expected
+    New color is focused, highlighted, selected and its RGB code is displayed.
+
+**Scenario 2:**
```

Comment:
This scenario does not work in IE8, where the color of selection beginning is selected inside the dialog. Additionally this scenario is not covered by unit tests.